### PR TITLE
Feature see guest responses

### PR DESCRIPTION
**Pull request by justinl-y**
Completes issue #25

### CODE REVIEW

File: js/scenes/HostPotluckGuestInvitesScreen/HostPotluckGuestInvitesScreen.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <View style={styles.rowContainer}>
+      {
+        Object.values(guestList).map((guest) => {
+          return (
+            <View style={styles.guestNameContainer} key={Math.random() * Date.now()}>
+              <Text style={styles.nameText}>{guest[0]}</Text>
+              <View style={styles.imageView}>
+                {(guest[1] === '+' && <Image source={inviteConfirmationStatus.YES.imageSource} />)
+                || (guest[1] === '-' && <Image source={inviteConfirmationStatus.NO.imageSource} />)
+                || <Image source={inviteConfirmationStatus.UNKNOWN.imageSource} />}
```
Comment (BlakeTurner) [inline js/scenes/HostPotluckGuestInvitesScreen/HostPotluckGuestInvitesScreen.js:17]: This is terrifying. It's nice to see that you understand short circuits, but this one goes pretty far!

You should write a function that takes a `guest` (or whatever `guest[1]` is) and returns an image component.

File: js/services/api/guest-invites.js [hunk 2]
```diff
@@ -112,29 +112,167 @@ const processSignUpEmailInvites = (userId, email) => {
...
+    if (key === 'userNoResponse' && inviteResponses[key].length > 0) {
+      inviteResponses[key].forEach((item) => {
+        guestResponses.push([`${item.firstName} ${item.lastName}`, '?']);
+      });
+    }
+    if (key === 'emailNoResponse' && inviteResponses[key].length > 0) {
+      inviteResponses[key].forEach((item) => {
+        guestResponses.push([item, '?']);
+      });
+    }
```
Comment (BlakeTurner) [inline js/services/api/guest-invites.js:241]: Suggest a refactor to write a function:

```js
const createGuestResponse = (key, item) => {
  const responseMap = {
    userAccept: [`${item.firstName} ${item.lastName}`, '+'],
    userReject: [`${item.firstName} ${item.lastName}`, '-'],
    ...etc
  }

  return responseMap[key];
}
```

And just throw in a guard clause to handle the `inviteResponses[key].length > 0` repetition.

File: js/services/api/guest-invites.js [hunk 3]
```diff
@@ -112,29 +112,167 @@ const processSignUpEmailInvites = (userId, email) => {
...
+  });
+
+  return guestResponses;
+};
+
+const getPotluckInviteResponses = (potluckId) => {
+  const promises = [
+    processtPotluckUserInviteResponses(potluckId, getGuestInviteUserDetailsAccept) || [],
```
Comment (BlakeTurner) [inline js/services/api/guest-invites.js:249]: Typo? processt

**Review by BlakeTurner — CHANGES_REQUESTED** [review]

File: js/scenes/HostPotluckGuestInvitesScreen/HostPotluckGuestInvitesScreen.js (hunk 1 above)
Comment (justinl-y) [reply js/scenes/HostPotluckGuestInvitesScreen/HostPotluckGuestInvitesScreen.js:17]: Done.

File: js/services/api/guest-invites.js (hunk 3 above)
Comment (justinl-y) [reply js/services/api/guest-invites.js:249]: Done.

**Review by BlakeTurner — APPROVED** [review]

File: js/services/api/guest-invites.js (hunk 2 above)
Comment (justinl-y) [reply js/services/api/guest-invites.js:241]: Done.

**Comment by justinl-y** [comment]
PR issues actioned.

**Pull request merged**